frontend/src/metabase/components/ToolbarButton/ToolbarButton.tsx

```diff
@@ -51,6 +52,7 @@ export const ToolbarButton = forwardRef(function ToolbarButton(
       onClick={handleButtonClick}
       bg={hasBackground ? undefined : "transparent"}
       disabled={disabled}
+      c="var(--mb-color-text-primary)"
```

Comment:
For some strange reason `c="text-dark"` did not work, and creating a `ToolbarButton.module.css` class and adding it here did not work. I have no idea why.